posthog/models/filters/mixins/property.py

```diff
@@ -51,6 +53,9 @@ def property_groups(self) -> PropertyGroup:
                 raise e
             except ValueError as e:
                 raise ValidationError(f"PropertyGroup is unparsable: {e}")
+        # already a PropertyGroup just return
+        elif isinstance(loaded_props, PropertyGroup):
+            return loaded_props
```

Comment:
I did add this logic for completeness so if you do pass in a PropertyGroup directly to the filter object, it will be handled correctly but it's not used for now